### ISSUE TITLE
Internal Server Error when clicking Preferences

### ISSUE THREAD

**Issue by x-tk** [searxng/searxng#731]
Hi there, really enjoy using this docker.  Have been running it for months now pretty flawlessly but as of the last week or so, when I click on "preferences", I get an internal server error.  This does sit behind an nginx proxy but that hasn't represented any issues previously and no configuration has been changed on my side.  Any insight into this problem?

**Comment by x-tk** [comment]
I should also add that I experience this error regardless of web browser and device that I use.

**Comment by not-my-profile** [comment]
Thanks for reporting this. It might be caused by #732 for which a fix is already on its way. To be sure we'd need more information about which exception occurred, which you should be able to find via some log. @return42 can probably further comment on that?

**Comment by return42** [comment]
@x-tk did you configured a engine that does not have an about property? Typical some of the offline engines might not have such a property.  You can verify it by adding an empty `about:` property.  Here is an example of such an offline engine:

```
  - name: locate
    categories : [ 'it', 'files' ]
    engine: command
    command: ['locate', '{{QUERY}}']
    shortcut: locate
    tokens: [ '123', '456' ]
    disabled: true
    # workaround issue #732
    about:
    delimiter:
        chars: ' '
        keys: ['line']
```

Can you confirm _workaround issue #732_ / adding `abbout:` ? .. if so, your issue is fixed by https://github.com/searxng/searxng/pull/733 and you can close this issue.  Otherwise please inform us.

**Comment by Onetryingtofollowthesunnah** [comment]
Same issue here as well, https://searx.be/preferences gives a 404

**Comment by return42** [comment]
@unixfox did you merged #733 or is it a different problem?

**Comment by unixfox** [comment]
> @unixfox did you merged #733 or is it a different problem?

I've my instance automatically updated to the latest commits every day at 12PM UTC.

**Comment by return42** [comment]
Do you know he reason for "Internal Server Error" https://searx.be/preferences

**Comment by unixfox** [comment]
This is fixed on https://searx.be

**Comment by return42** [comment]
@x-tk this issue is fixed by #733 / I close this issue.

**Comment by Onetryingtofollowthesunnah** [comment]
> @unixfox did you merged #733 or is it a different problem?

I didnt merge or did anything
Regardless, its working now, Thanks!
